src/decorator/decoratorconfig.ts

```diff
@@ -28,7 +28,7 @@ const getEnvironment = (): Environment => {
   } else if (erDev()) {
     return "q2";
   } else {
-    return "local";
+    return "mock";
```

Comment:
Endret denne for å bli kvitt feilmeldingen øverst som gjelder tilkobling til contextholderen lokalt.